tests/accel/upgrade/upgradeconfig/config.go

```diff
@@ -0,0 +1,45 @@
+package upgradeconfig
```

Comment:
You need to create ./tests/accel/internal/accelinittools as well as ./tests/accel/internal/accelconfig on level above.

Reply:
Done, thanks